configs/sst_kernel_rats-kernel-rt-base.yaml

```diff
@@ -12,6 +12,8 @@ data:
         - tuned-profiles-nfv-host-bin
     labels:
         - eln
+    limit_arches:
```

Comment:
This won't work. If you want to make all the package s above to only be on x86_64, you can move them under:

```
 arch_packages:
     x86_64:
       - tuned-profiles-realtime
       ...
```

and leave the packages field empty:

```
packages: []
```

https://github.com/minimization/content-resolver/blob/master/config_specs/workload.yaml#L61